ms-common-impl/src/main/java/net/trajano/ms/vertx/beans/JwksProvider.java

```diff
@@ -1,37 +1,33 @@
 package net.trajano.ms.vertx.beans;
 
-import static net.trajano.ms.core.Qualifiers.JWKS_CACHE;
-
-import java.security.KeyPair;
-import java.security.KeyPairGenerator;
-import java.security.interfaces.RSAPrivateKey;
-import java.security.interfaces.RSAPublicKey;
-import java.text.ParseException;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Random;
-
-import javax.annotation.PostConstruct;
-
+import net.trajano.ms.core.CryptoOps;
+import org.jose4j.jwa.Algorithm;
+import org.jose4j.jwk.*;
+import org.jose4j.jws.AlgorithmIdentifiers;
+import org.jose4j.jws.JsonWebSignature;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.jose4j.jws.JsonWebSignature'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)